Lombiq.Tests.UI/Models/InstanceCommandLineArgs.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections.Generic;
+
+namespace Lombiq.Tests.UI.Models;
+
+public class InstanceCommandLineArgumentsBuilder
+{
+    private readonly List<string> _arguments = new();
+
+    public InstanceCommandLineArgumentsBuilder AddSwitch(string argument)
+    {
+        _arguments.Add($"{PrepareArg(argument)}");
+
+        return this;
+    }
+
+    public InstanceCommandLineArgumentsBuilder AddWithValue<T>(string key, T value)
+    {
+        _arguments.Add($"{PrepareArg(key)}={value}");
+
+        return this;
+    }
+
+    private static string PrepareArg(string argument)
+    {
+        if (!argument.StartsWith("--", System.StringComparison.InvariantCultureIgnoreCase))
```

Comment:
This handles `--argument`, but not `-argument` which is also valid. So just doing `.TrimStart('-')` would be better.